src/cephadm/cephadm

```diff
@@ -483,6 +483,16 @@ class CephIscsi(object):
                   "umount {0}; fi".format(mount_path)
         return cmd.split()
 
+    def get_tcmu_runner_container(self):
+        # type: () -> CephContainer
+        tcmu_container = get_container(self.fsid, self.daemon_type, self.daemon_id)
+        tcmu_container.entrypoint = "/usr/bin/tcmu-runner"
+        tcmu_container.args.append("&")
```

Comment:
does this belong into the bash script? I mean, this is not part of the continer args, right?

Reply:
yeah, that's probably cleaner. I'll make the change :)